UWP/App.cpp

```diff
@@ -235,9 +252,10 @@ void App::OnSuspending(Platform::Object^ sender, SuspendingEventArgs^ args) {
 	auto app = this;
 
 	create_task([app, deferral]() {
+		g_Config.Save("App::OnSuspending");
 		app->m_deviceResources->Trim();
-		deferral->Complete();
-	});
+	    deferral->Complete();
```

Comment:
Spaces and tabs mixed here.  I'm not sure what IDE you're using, but look for a plugin that enables EditorConfig.  It'll make it easier not to use the wrong indentation style across projects - even when they have different rules.

Also, please prefer:
```c++
create_task([app, deferral]() {
	asdf();
	bar();
});
```

Over:

```c++
create_task([app, deferral]() {
	asdf();
	bar();
	});
```

-[Unknown]